advocate_europe/assets/scss/components/_blocks.scss

```diff
@@ -125,9 +125,26 @@
     }
 
     .blockblog-image {
+        background-size: cover;
+        background-repeat: no-repeat;
+        display: block;
+        position: relative;
+        box-sizing: content-box;
+        width: 100%;
+        height: auto;
```

Comment:
This should be the default value anyway.

Reply:
No, doesn't seem to be. Removing it messes up the shadow in the idea space. Well, it might actually still be the default here in the blog block anyway, but is needed in the ideaspace.